tests/ord-aggregator/tests/handler_test.go

```diff
@@ -89,6 +87,12 @@ const (
 	documentationLabelSecondValue = "With multiple values"
 )
 
+var (
+	// The expected number is increased with initial number of global vendors/products before test execution
+	expectedNumberOfProducts = 7
+	expectedNumberOfVendors  = 7
+)
```

Comment:
There should be 6 as there is 1 global product and 1 global vendor in the local setup.

If the test fails with 6 I suspect that the getting of global vendors & products is the problem. Otherwise, it will fail again on real env.